example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+# filepath: eg:  /home/purnendu/Development/talawa-api/example/linux/systemd/Talawa-api.sh 
+
+# Load nvm and use the correct Node.js version
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Shellcheck (0.10.0)

[info] Not following: ./nvm.sh was not specified as input (see shellcheck -x).

(SC1091)